settings.gradle

```diff
@@ -1,3 +1,23 @@
+pluginManagement {
+    resolutionStrategy {
+        eachPlugin {
+            // Work around https://github.com/gradle/gradle/issues/1697.
+            if (requested.version == null) {
+                def versionPropertyName = (requested.id.id == 'org.jetbrains.kotlin.jvm') ?
+                        "kotlinVersion" : "${requested.id.name}Version"
+                println("Checking for plugin version property '$versionPropertyName'.")
```

Comment:
Seems verbose to print this all the time, what about `logger.info("...")` instead?

Reply:
I'm fine with using `logger`, will do. About `detektPluginVersion`, yes, that's what I mean. Even if we could also otherwise visuall separate plugin versions from "normal" versions in `settings.gradle` it might still be useful to understand they refer to plugins. I'll make that change.